docs/developer/base/http.md

```diff
@@ -0,0 +1,54 @@
+# HTTP
+
+-----
+
+Whenever you need to make HTTP requests, the base class provides a convenience member that has the same interface as the
+popular [requests][requests-github] library and ensures consistent behavior across all integrations.
+
+The wrapper automatically parses and uses configuration from the `instance`, `init_config`, and Agent config. Also, this
+is only done once during initialization and cached to reduce the overhead of every call.
+
+All you have to do is e.g.:
```

Comment:
Just a note about "just do X" / "doing Y is simple" / "doing Z is easy" and that kind of phrasing - it might look obvious and "simple" but it's not actually helpful and might actually be counter-productive if the user is just getting started with the wrapper. (I've seen efforts of reducing this kind of phrasing in eg the Django docs and I think it's a nice thing to pursue.) I haven't seen many such phrasing in these docs but just taking the opportunity here to explicitly acknowledge this is prob a good habit to pursue :)

Eg here we could use a more neutral:

```suggestion
For example, to make a GET request you would use:
```